types/Address/Address.d.ts

```diff
@@ -128,15 +129,63 @@ export declare class Address implements IAddress {
    *    If verification fails, ask them to double check their input; if they confirm that their data is correct, assume they know their address more correctly than the verification process.
    *
    * @see https://www.easypost.com/docs/api/node#create-and-verify-addresses
+   *
+   * @param {Object} params The parameters to create an {@link Address} with.
+   * @returns {Promise<Address>} The created and verified {@link Address}.
+   */
+  public create(params: Object): Promise<Address>;
+
+  /**
+   * Retrieve a list of all Addresses.
+   *
+   * The Address List is a paginated list of all {@link Address} objects associated with the given API Key.
+   * It accepts a variety of parameters which can be used to modify the scope.
+   * The has_more attribute indicates whether additional pages can be requested.
+   * The recommended way of paginating is to use either the before_id or after_id parameter to specify where the next page begins.
+   *
+   * @see https://www.easypost.com/docs/api/node#retrieve-a-list-of-addresses
+   *
+   * @param {Object} params - The parameters to use for the request.
+   */
+  static all(params?: IAddressListParameters): Promise<{ addresses: Address[]; has_more: boolean }>;
+
+  /**
+   * Create and verify a new {@link Address} in one API call.
+   *
+   * Verify an {@link Address}
+   *  - Verifying an {@link Address} before you ship is a great way to reduce issues with delivery.
+   *    Creating a verified Address is as simple as including an enumerated list of the verifications you'd like EasyPost to perform in the `verify` or `verify_strict` url parameters.
+   *    If any of the verification checks included in the verify_strict list fail an error will be returned from the API.
```

Comment:
```suggestion
   *    If any of the verification checks included in the `verify_strict` list fail an error will be returned from the API.
```

All places in this PR should use backticks around "code" words, such as parameters, anything snake_cased, etc.